src/chains/supportedChains.svm.int.spec.ts

```diff
@@ -52,8 +64,8 @@ describe.concurrent('SVM chains block explorer check', () => {
     { timeout: 10_000, retry: 3 },
     async ({ blockExplorerUrl }) => {
       const response = await fetch(blockExplorerUrl)
-      expect(response.url).toBe(blockExplorerUrl)
-      expect(response.ok).toBe(true)
+      expect(isSameUrl(blockExplorerUrl, response.url)).toBeTruthy()
+      expect(response.ok).toBeTruthy()
```

Comment:
hmm what is `ok` in that case? Because `'false'` is truthy, so do we really want this behaviour?

Reply:
Solana explorers should pass tests fine, there were some issues with Bitcoin ones. Would you like to remove these checks completely?

Reply:
yes, I wanna remove them completely, and just ensure the rpc and explorer array are not empty. 
Normally, we add multiple rpcs, some of them may be invalid, or we are blocked. Same to explorer, the explorer could refuse our request, which are unstable.
what do you think of it?